MicrosoftTeams.schema.json

```diff
@@ -66,6 +66,12 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
+                // the below property can be avoided by reducing the size of the short property.
+                "tiny": {
+                    "type": "string",
+                    "description": "A short display name for the app.",
```

Comment:
A short...er display name for the app =P

Reply:
Introducing this field is fine... I just meant change the description to be a bit more descriptive about what this name variation is for.. short is already a short display name so maybe make it something like tiny name for where space is very limited.